src/coreclr/inc/jiteeversionguid.h

```diff
@@ -43,11 +43,11 @@ typedef const GUID *LPCGUID;
 #define GUID_DEFINED
 #endif // !GUID_DEFINED
 
-constexpr GUID JITEEVersionIdentifier = { /* 77b6df16-d27f-4118-9dfd-d8073ff20fb6 */
-    0x77b6df16,
-    0xd27f,
-    0x4118,
-    {0x9d, 0xfd, 0xd8, 0x7, 0x3f, 0xf2, 0xf, 0xb6}
+constexpr GUID JITEEVersionIdentifier = { /* ece2fd20-fd72-4602-ba1b-d57dff8a692f */
+    0xece2fd20,
+    0xfd72,
+    0x4602,
+    {0xba, 0x1b, 0xd5, 0x7d, 0xff, 0x8a, 0x69, 0x2f}
```

Comment:
😢

Reply:
Are you referring to the case that once it's merged SPMI is not available for 6h? In this case we can merge them in batch/or only on weekends

Reply:
Yeah, we have a few upcoming JIT-EE GUID changes and it seems superpmi-collect is not super robust so it can be somewhat disruptive. Will be nice if we can try to batch them.

Reply:
With that said, we should also ensure we don't get into a mode where we're "afraid" of taking JIT-EE GUID changes. E.g. by improving superpmi-collect or finding ways to make superpmi-diffs better handle the situation.

Reply:
Reverted guid update, but I share your concerns 🙂